src/App.tsx

```diff
@@ -15,8 +15,28 @@ import { SettingsPage } from "./pages/settings"
 import { UserProfilePage } from "./pages/user-profile"
 import { ViewOrderPage } from "./pages/view-order"
 import { ViewSnippetPage } from "./pages/view-snippet"
+import { useEffect, useState } from "react"
+
+// Define the type for a snippet
+interface Snippet {
+  code: string
+  // Add other properties as needed
+}
 
 function App() {
+  const [snippets, setSnippets] = useState<Snippet[]>([])
+
+  useEffect(() => {
+    // Fetch snippets from the API instead of loading them directly
+    if (import.meta.env.DEV) {
+      fetch("/api/snippets")
+        .then((response) => response.json())
+        .then((data) => {
+          setSnippets(data.snippets)
+        })
```

Comment:
no frontend code should be changed for this PR, it is an entirely backend change